cmd/flags.go

```diff
@@ -118,6 +118,10 @@ func CreateFlags(defaultPath string) []cli.Flag {
 			Usage: "Set the DNS timeout value to a specific value in seconds. Used only when performing authoritative name server queries.",
 			Value: 10,
 		},
+		&cli.BoolFlag{
+			Name:  "bupem",
+			Usage: "Generate an additional 'bottom-up' .bupem (PEM, base64) file by concatenating the .key, .crt and issuer .crt together, in that order.",
```

Comment:
"PEM, base64" is redundant, as all PEM files encode X.509 DER serialization data as Base64.

```suggestion
			Usage: "Generate an additional 'bottom-up PEM' file (.bupem) by concatenating the private key, leaf certificate and issuer chain (in that order) into a single file.",
```

Reply:
Not strictly necessary - abstract is equally intelligible. First 'the' is acceptable in native English.